### PR TITLE
PR for initial commit for Online Pharmacy Project.

### PR DESCRIPTION
**Pull request by shreyasinha12**
PR for initial commit for Online Pharmacy Project.'

Please review and merge

### CODE REVIEW

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/Login.jsp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	position: absolute;
+	left: 650px
+}
+</style>
+<body
+	background="C:\Users\shreyasin\workspace\OnlinePharmacyNew\WebContent\WEB-INF\views\images\images (3).jpg">
+	<div align="center">
+		<form action="/OnlinePharmacyNew/loginsuccess" method="post"
+			onsubmit="return validate()">
+			<!-- ModelAttribute="login" -->
```
Comment (shreyasinha12) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/Login.jsp]: remove comments

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/Login.jsp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		}
+		var pswrd = document.getElementById("password").value;
+		if (pswrd == "") {
+			alert("please enter password");
+		}
+	}
+</script>
+<style>
+body {
+	background-repeat: no-repeat;
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/Login.jsp]: If possible, Please create common css file use them in the jsps instead directly writing them in jsps

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/Login.jsp [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+	background-size: 1400px 900px;
+}
+
+#b1 {
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/Login.jsp]: fyi

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/LoginFailed.jsp [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<style>
+body
+{
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/LoginFailed.jsp]: FYI

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/LoginFailed.jsp [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+background-repeat: no-repeat;
+background-size: 1400px 900px;
+}
+#b1
+{
+position:absolute;
+left:650px
+}
+</style>
+<body background="C:\Users\shreyasin\workspace\OnlinePharmacyNew\WebContent\WEB-INF\views\images\images (3).jpg">
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/LoginFailed.jsp]: What if i am deploying this application in another machine/server.

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/MedicineFailed.jsp [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+background-repeat: no-repeat;
+background-size: 1400px 900px;
+}
+#b1
+{
+position:absolute;
+left:650px
+}
+</style>
+<body background="C:\Users\shreyasin\workspace\OnlinePharmacyNew\WebContent\WEB-INF\views\images\images (3).jpg">
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/MedicineFailed.jsp]: fyi

Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/MedicineFailed.jsp]: find an alternate way to use this jpg file

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/WelcomePage.jsp [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+<h1  style="text-align:center;">NetMeds Online Pharmacy</h1>
+<style>
+body {
+
+/*  background-image:url("C:\Users\shreyasin\workspace\OnlinePharmacy\WebContent\imgpic.jpg"); */
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/WelcomePage.jsp]: remove this commented code if not needed

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/WelcomePage.jsp [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+margin-bottom:10px; /*or whatever you want the spacing to be
+} 
+/* #footer {
+ width:100%;
+   position:absolute;
+   bottom:0;
+   } */
+</style>
+</head>
+<body background="C:\Users\shreyasin\workspace\OnlinePharmacyNew\WebContent\WEB-INF\views\images\online-pharmacy-safe.jpg" >
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/WelcomePage.jsp]: fyi

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/afterAddingMedicine.jsp [hunk 9]
```diff
@@ -0,0 +1,15 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Insert title here</title>
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/afterAddingMedicine.jsp]: mention appropriate title

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/registerationSuccess.jsp [hunk 10]
```diff
@@ -0,0 +1,36 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<style>
+body {
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/registerationSuccess.jsp]: fyi

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/searchFile.jsp [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<style>
+body{
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/searchFile.jsp]: fyi

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/showMedicine.jsp [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+background-size: 1400px 900px;
+/* font-style:arial;
+font-weight: bold;
+font-size: 16px; */
+
+}
+</style>
+
+
+<body   background="C:\Users\shreyasin\workspace\OnlinePharmacyNew\WebContent\WEB-INF\views\images\ban1.jpg">>
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/showMedicine.jsp]: fyi

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/uploadPrescription.jsp [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+<style>
+body{
+background-repeat: no-repeat;
+background-size: 1500px 1000px;
+font-style:arial;
+font-weight: bold;
+font-size: 16px;
+}
+</style>
+<body background="C:\Users\shreyasin\workspace\OnlinePharmacyNew\WebContent\WEB-INF\views\images\download (1).jpg">
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/uploadPrescription.jsp]: fyi

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Controllers/RegisterationController.java [hunk 14]
```diff
@@ -0,0 +1,61 @@
...
+			registerObj.setPassword(registerBean.getPassword());
+			registerObj.setLastName(registerBean.getLastName());
+			registerObj.setRoleField(registerBean.getRoleField());	
+			registerObj.setUsername(registerBean.getUsername());
+			registerObj.setLocality(registerBean.getLocality());
+			return registerObj;
+		}
+		
+		
+		/*@RequestMapping(value ="/Login", method = RequestMethod.POST)
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Controllers/RegisterationController.java]: remove if not needed

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Model/MedicineInformation.java [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+	@Entity
+	@Table(name="MedicineInformation")
+	public class MedicineInformation {
+	//	@Override
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Model/MedicineInformation.java]: fyi

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Service/PasswordUpdateService.java [hunk 16]
```diff
@@ -0,0 +1,6 @@
+package com.OnlinePharmacy.Service;
+
+
+public interface PasswordUpdateService {
+	public void updatePassword(String username,String newpassword,String confirmpassword);
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Service/PasswordUpdateService.java]: doc comments?

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Service/MedicineInformationService.java [hunk 17]
```diff
@@ -0,0 +1,16 @@
+package com.OnlinePharmacy.Service;
+
+import java.util.List;
+
+import com.OnlinePharmacy.Model.MedicineInformation;
+
+public interface MedicineInformationService {
+	
+
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Service/MedicineInformationService.java]: Documentation comments?

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Service/MedicineInformationService.java [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+import java.util.List;
+
+import com.OnlinePharmacy.Model.MedicineInformation;
+
+public interface MedicineInformationService {
+	
+		
+		public void addMedicine(MedicineInformation med);
+		//public List<MedicineInformation> searchMedicine(MedicineInformation med);
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Service/MedicineInformationService.java]: remove this commented line

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/DAOIMPL/RegisterationDAOIMPL.java [hunk 19]
```diff
@@ -0,0 +1,33 @@
...
+import com.OnlinePharmacy.DAO.RegisterationDAO;
+import com.OnlinePharmacy.Model.Registration;
+
+@Repository
+public class RegisterationDAOIMPL implements RegisterationDAO {
+	@Autowired
+	private SessionFactory sessionFactory;
+	
+/*	@Autowired
+	private DataSource dataSource;*/
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/DAOIMPL/RegisterationDAOIMPL.java]: fyi. commented code

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/DAOIMPL/PasswordUpdate.java [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+	        	Query query = session
+                        .createQuery("update Registration set Password=:n where username=:i");
+           query.setString("n", newpassword);
+           query.setString("i", username);
+           query.executeUpdate();
+
+
+	        }
+	        else{
+	               System.out.println("password is not matching");
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/DAOIMPL/PasswordUpdate.java]: not recommended to use system.out.println statements.


Please implement. log4j framework for the logger messages on console

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/showMedicineDetails.jsp [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+background-size: 1400px 900px;
+/* font-style:arial;
+font-weight: bold;
+font-size: 16px; */
+
+}
+</style>
+
+
+<body   background="C:\Users\shreyasin\workspace\OnlinePharmacyNew\WebContent\WEB-INF\views\images\ban1.jpg">>
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/showMedicineDetails.jsp]: what if we deploying this code in server/another machine? will this path available?

File: Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/updatePassword.jsp [hunk 22]
```diff
@@ -0,0 +1,76 @@
...
+position:absolute;
+left:689px
+}
+#b2
+{
+position:absolute;
+left:689px
+}
+</style>
+<body  background="C:\Users\shreyasin\workspace\OnlinePharmacyNew\WebContent\WEB-INF\views\images\forgot-password-php.jpg">
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/WebContent/WEB-INF/views/updatePassword.jsp]: fyi

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Controllers/LoginController.java [hunk 23]
```diff
@@ -0,0 +1,111 @@
...
+					loginBean.getUsername(), loginBean.getPassword());
+			String getRole = loginService.getRole(
+					loginBean.getUsername());
+
+
+			if (isValidUser && getRole.equals("user_normal"))
+
+			{
+
+				System.out.println("User Login Successful");
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Controllers/LoginController.java]: fyi. implement loggers

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Controllers/MedicineInformationController.java [hunk 24]
```diff
@@ -0,0 +1,147 @@
...
+              
+     
+		this.medicineService.addMedicine(med);
+                return new ModelAndView("afterAddingMedicine");
+	}
+	
+	@RequestMapping(value= "/afterAddingMedicine", method = RequestMethod.POST)
+    public ModelAndView afterAddingMedicine(@ModelAttribute("MedicineInformation") MedicineInformation med,BindingResult result,HttpSession session){
+               
+          System.out.println("MedicineName...."+med.getMedicineName());
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Controllers/MedicineInformationController.java]: remove this system.out.println statements and implement loggers

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Controllers/MedicineInformationController.java [hunk 25]
```diff
@@ -0,0 +1,147 @@
...
+
+
+
+
+
+@RequestMapping(value="/searchFile", method=RequestMethod.POST)
+public ModelAndView searchMethod(@ModelAttribute("MedicineInformation") MedicineInformation medicine,BindingResult result, HttpServletRequest request,HttpSession session){
+	session.getAttribute("username");
+	String med = medicine.getMedicineName();
+	/*medicineService.search(med);*/
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/Controllers/MedicineInformationController.java]: remove the commented code if you not needed

File: Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/DAO/MedicineInformationDAO.java [hunk 26]
```diff
@@ -0,0 +1,16 @@
+package com.OnlinePharmacy.DAO;
+
+
+	import java.util.List;
+
+import com.OnlinePharmacy.Model.MedicineInformation;
+
+	public interface MedicineInformationDAO {
+	void addMedicine(MedicineInformation med);
```
Comment (rameshkum) [inline Shreya_Sinha/OnlinePharmacyNew/src/com/OnlinePharmacy/DAO/MedicineInformationDAO.java]: add documentation comments for all the methods and Java files as well

**Pull request merged**